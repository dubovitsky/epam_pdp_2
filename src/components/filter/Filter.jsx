import React from 'react';

const Filter = () => {
    return (
        <div className="text-right">
            <span>Sort by: </span>
            <a href="#">realease date</a>/
            <a href="#">rating</a>
        </div>
    )
}

export default Filter;