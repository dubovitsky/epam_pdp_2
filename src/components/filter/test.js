import React from 'react';
import { shallow } from 'enzyme';
import Filter from './Filter';

describe('Filter', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Filter debug />);
  
    expect(component).toMatchSnapshot();
  });
});