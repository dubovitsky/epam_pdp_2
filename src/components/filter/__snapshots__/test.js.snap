// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter should render correctly in "debug" mode 1`] = `
<div
  className="text-right"
>
  <span>
    Sort by: 
  </span>
  <a
    href="#"
  >
    realease date
  </a>
  /
  <a
    href="#"
  >
    rating
  </a>
</div>
`;
